components/brave_today/browser/brave_news_controller.cc

```diff
@@ -696,4 +687,19 @@ void BraveNewsController::MaybeInitPrefs() {
   }
 }
 
+void BraveNewsController::OnPublishersUpdated(
```

Comment:
IIUC, NTP gets publishers info when news is enabled via this callback.
W/o this info, NTP crashed while news feed customzing page is rendered.
I'm curious is there a race condition between the customizing page rendering and this info delivery timing.